test/integration/targets/win_psmodule/tasks/main.yml

```diff
@@ -87,6 +87,7 @@
       - "repo_fail|failed"
       - "module_check.stdout_lines[0] == '{{ powershell_module }}'"
       - "allow_clobber_check.stdout_lines[0] == '{{ allow_clobber_module }}'"
+  when: version.stdout_lines[0]|int >= 5
```

Comment:
I would use the `ansible_powershell_version` fact here.

And put everything in a large block:
```yaml
- name: Perform integration tests for Powershell 5+
  when: ansible_powershell_version >= 5
  block:
  - win_psmodule: ...
  - win_psmodule: ...
  - assert: ...
`